src/ansys/geometry/core/designer/design.py

```diff
@@ -717,7 +715,9 @@ def delete_named_selection(self, named_selection: NamedSelection | str) -> None:
             removal_id = named_selection.id
 
         self._grpc_client.log.debug(f"Named selection {removal_name} deletion request received.")
-        self._named_selections_stub.Delete(EntityIdentifier(id=removal_id))
+        self._grpc_client.services.named_selection.delete_named_selection(
+            id=EntityIdentifier(id=removal_id)
+        )
```

Comment:
The EntityIdentifier creation should happen inside the ``delete_named_selection`` method. We shouldn't expose any gRPC object outside of the "_grpc" subpackage